scripts/lib/google-sheets/incentives-sheet-exporter.ts

```diff
@@ -91,24 +92,62 @@ function createCellValue(
   return valToWrite;
 }
 
+function createCellDataValidation(
+  col: string,
+  fieldsWithEnums: FieldMetadata[],
+): sheets_v4.Schema$DataValidationRule | undefined {
+  for (const [ind, field] of fieldsWithEnums.entries()) {
+    if (col === field.column_aliases[0]) {
+      const columnLetter = colToLetter(ind * 3 + 1);
+      const lastEnumRow = Object.keys(field.values!).length + 1;
+      return {
+        strict: true,
+        // Make Google Sheets to show a dropdown UI element. Unfortunately,
+        // it's not possible in the API to force the "chip" version, just a
+        // regular list.
```

Comment:
I can't see this in the sample sheet because I only have view access, but I believe you that it works

Reply:
I changed permissions to editor in case you want to see it later.